content/installation/installing-manager.md

```diff
@@ -108,7 +108,7 @@ Emptying the Cloudify Manager database is irreversible.
 
 {{% /gsWarning %}}
 
-During both installation and configuration of Cloudify Manager, you can use the ```--clean-db``` option to empty all of the data from the Cloudify Manager database.
+During both installation and configuration of Cloudify Manager, you can use the ```--clean-db``` option to empty all of the data from the Cloudify Manager database. The first installation always assumes the clearing of the DB, so it is not necessary to pass the `--clean-db` flag on the initial install.
```

Comment:
The initial installation does not require the `--clean-db` flag.

Reply:
I know, this is what I'm trying to say here. I didn't see it mentioned anywhere else

Reply:
I mean to replace the sentence that you added with: The initial installation does not require the --clean-db flag.

Reply:
Much better :)